test-result-summary-client/src/PerfCompare/lib/JenkinsRunJSON.js

```diff
@@ -65,13 +61,13 @@ export default class JenkinsRunJSON {
             for (let z = 0; z < curVariantObject.metrics.length; z++) {
                 if (Array.isArray(curVariantObject.metrics[z].value)) {
                     try {
-                        curVariantObject.metrics[z]["mean"] = math.mean(curVariantObject.metrics[z].value);
+                        curVariantObject.metrics[z]["mean"] = curVariantObject.metrics[z].value.mean;
                     } catch(e) {
                         curVariantObject.metrics[z]["mean"] = null;
                     }
 
                     try {
-                        curVariantObject.metrics[z]["ci"] = BenchmarkMath.confidence_interval(curVariantObject.metrics[z].value) * 100;
+                        curVariantObject.metrics[z]["ci"] = curVariantObject.metrics[z].value.CI * 100;
```

Comment:
`ci` doesn't seem to be used in Perf Compare. We are directly using the stored CI value in this line: 
`curBaselineCI = math.round((this.state.benchmarkRuns.benchmarkRunBaseline.parsedVariants[i].metrics[j].value.CI), 3);`